docs/csharp/tutorials/pattern-matching.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: Use pattern matching features to extend data types
+title: "Tutorial: pattern matching to return a value"
```

Comment:
Same comment as above on the title.  Thoughts?

Reply:
How about this? data-driven I think pushes the length of the title too far.
```suggestion
title: "Tutorial: Build algorithms with pattern matching"
```